Sources/ACKategories/SwiftUIExtensions/View+endEditingOnTap.swift

```diff
@@ -0,0 +1,22 @@
+#if canImport(UIKit) && !os(watchOS)
+import SwiftUI
+
+@available(iOS 13.0, *)
+public extension View {
+
+    @available(iOSApplicationExtension, unavailable)
```

Comment:
We can move this availability clause to the extension itself so it is at one place.